translations/messages.en.yaml

```diff
@@ -2,6 +2,9 @@
 # (c) Paweł Jędrzejewski
 
 sylius_price_history:
+    form:
```

Comment:
I would be for adding it to `ui` section instead of `form`, but I have no strong arguments for that, so 🤷🏻

Reply:
I just did it as it is on other plugins. I don't have other strong arguments 😄